includes/Jobs/RequestWikiRemoteAIJob.php

```diff
@@ -0,0 +1,395 @@
+
+		$this->id = $params['id'];
+		$this->reason = $params['reason'];
+		$this->sitename = $params['sitename'];
+		$this->subdomain = $params['subdomain'];
+	}
+
+	public function run(): bool {
+		$services = MediaWikiServices::getInstance();
+
+		if ( !$this->config->get( ConfigNames::OpenAIConfig )['apikey'] ) {
+			$this->logger->debug( "OpenAI API key is missing! AI job cannot start." );
+			throw new ConfigException( 'OpenAI API key is missing! Cannot query API without it!' );
+		} elseif ( !$this->config->get( ConfigNames::OpenAIConfig )['assistantid'] ) {
+			$this->logger->debug( "OpenAI Assistant ID is missing! AI job cannot start." );
+			throw new ConfigException( 'OpenAI Assistant ID is missing! Cannot run AI model without an assistant!' );
+		}
+
+		$this->wikiRequestManager->loadFromID( $this->id );
+		$this->logger->debug( "Loaded request {$this->id} for AI approval." );
+
+		if ( !$this->canAutoApprove() ) {
+			$this->logger->debug( "Wiki request {$this->id} was not auto-evaluated due to denylist." );
+			return true;
+		}
+
+		// Initiate OpenAI query for decision
+		$this->logger->info( "Querying OpenAI for decision on wiki request {$this->id}..." );
+		$apiResponse = $this->queryOpenAI( $this->sitename, $this->subdomain, $this->reason );
+
+		if ( !$apiResponse ) {
+			return true;
+		}
+
+		// Extract response details with default fallbacks
+		$outcome = $apiResponse['recommendation']['outcome'] ?? 'reject';
+		$comment = $apiResponse['recommendation']['public_comment'] ?? 'No comment provided. Please check logs.';
+
+		$this->logger->info( "AI decision: {$outcome} for request {$this->id}. Comment: {$comment}" );
+
+		if ( $this->config->get( ConfigNames::OpenAIConfig )['dryrun'] ) {
+			return $this->handleDryRun( $outcome, $comment );
+		} else {
+			return $this->handleLiveRun( $outcome, $comment );
+		}
+	}
+
+	private function handleDryRun( string $outcome, string $comment ): bool {
+		$services = MediaWikiServices::getInstance();
+		$outcomeMessage = Message::newFromKey( "requestwikiqueue-$outcome" )->text();
+		$commentText = Message::newFromKey( 'requestwiki-ai-decision-dryrun' )
+		->params( $outcomeMessage, $comment )
+		->inLanguage( $services->getContentLanguage() )
+		->text();
+
+		$this->wikiRequestManager->addComment(
+			comment: $commentText,
+			user: User::newSystemUser( 'CreateWiki AI' ),
+			log: true,
+			type: 'comment',
+			notifyUsers: []
+		);
+
+		$dryRunMessages = [
+			'approve' => "Wiki request {$this->id} was approved by AI but not automatically created.",
+			'moredetails' => "Wiki request {$this->id} needs revision but was not automatically marked.",
+			'decline' => "Wiki request {$this->id} was declined by AI but not automatically marked.",
+			'onhold' => "Wiki request {$this->id} requires manual review.",
+		];
+
+		$this->logger->debug(
+			"DRY RUN: " . ( $dryRunMessages[$outcome] ?? "Unknown outcome for request {$this->id}." ),
```

Comment:
single quotes and remove {$this->id} from this and all the $dryRunMessages. Just use {id} which will get it from the options in logger.